ipa-core/src/ff/prime_field.rs

```diff
@@ -433,6 +433,25 @@ mod fp32bit {
 mod fp61bit {
     field_impl! { Fp61BitPrime, u64, u128, 61, 2_305_843_009_213_693_951 }
 
+    impl Fp61BitPrime {
+        #[must_use]
+        pub const fn const_truncate(input: u64) -> Self {
+            Self(input % Self::PRIME)
+        }
+
+        #[must_use]
+        pub const fn const_neg(input: Self) -> Self {
+            Self(Self::PRIME - input.0)
+        }
```

Comment:
This is a constant function and needed when declaring constants. Since I am using some field elements like minus 2 or minus one half quite frequently, I don't want to recompute them every time but rather define them as constants